common/exception/ErrorMessage.java

```diff
@@ -184,6 +186,8 @@ public static void loadConstants() {
                 new Session(2, "Attempted to open a transaction from closed session.");
         public static final Session SCHEMA_ACQUIRE_LOCK_TIMEOUT =
                 new Session(3, "Could not acquire lock for schema session. Another schema session may have been left open.");
+        public static final Session SESSION_IDLE_TIMEOUT_NOT_CONFIGURABLE =
+                new Session(4, "The session idle timeout is not configurable at the '%s' level.");
```

Comment:
added a new error if someone tries to set a session idle timeout at the tx/query level, which is nonsense